src/WebManager/app/src/services/docker.ts

```diff
@@ -66,24 +98,59 @@ export class Docker {
         let config: AxiosRequestConfig = {
             cancelToken: cancel,
             baseURL: this.registryEndpoint,
-            params: { },
+            params: {},
             headers: {
                 "Registry": this.registryName,
                 "Accept": "application/vnd.docker.distribution.manifest.v2+json; 0.6, " +
-                    "application/vnd.docker.distribution.manifest.v1+json; 0.5",
+                "application/vnd.docker.distribution.manifest.v1+json; 0.5," +
+                "application/vnd.docker.distribution.manifest.list.v2+json; 0.7",
```

Comment:
They mean the weight of the option, so first it would ask for the manifest.list, if that doesn't exist then it would ask for a regular manifest.v2 and if that doesn't exist it would ask for the manifest.v1